website/docs/add-secure-apps/providers/oauth2/fontchannel_and_backchannel_logout.mdx

```diff
@@ -0,0 +1,117 @@
+---
+title: OAuth2/OpenID Connect front-channel and back-channel logout
+description: Configure front-channel and back-channel logout for OAuth2/OpenID Connect providers
+authentik_version: "2025.8.0"
+authentik_preview: true
+---
+
+## Overview
+
+OAuth2/OpenID logout is a security feature defined in the OpenID Connect specification. It allows an OpenID Provider (OP), such as authentik, to notify Relying Parties (RPs) when a user session ends. This ensures that all associated applications can properly terminate the user's session.
+
+For more information about single logout across all providers, see the [Single Logout (SLO) Overview](../../flows-stages/stages/single_logout.md).
```

Comment:
This link will have to be updated when you move the SLO docs to be under Providers.